### PR TITLE
Fix unimplemented message on reload

### PR DESCRIPTION
**Pull request by tylerconstance**
Fixed an issue where reloading a route would throw an `unimplemented` message. Created a `NoMatch` Component to display as a 404 page

### CODE REVIEW

File: jsx/components/CouncilorDetail.jsx [hunk 1]
```diff
@@ -3,19 +3,25 @@ import PropTypes from 'prop-types';
...
       const slug = slugify(a.name);
-      if (slug === cid) {
-        return a;
-      }
-      return null;
+      return (slug === cid);
     });
 
+    if (!councilor) {
+      return <Redirect to="/404" push={true} />
```
Comment (tylerconstance) [inline jsx/components/CouncilorDetail.jsx]: The `to` destination here is arbitrary; let me know if there's something that makes more sense!

Comment (sethwklein) [reply jsx/components/CouncilorDetail.jsx]: I'd love to have it not redirect at all. That way it doesn't throw out the information the visitor had stored in the URL.

File: jsx/components/NoMatch.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const NoMatch = () => {
+  return (
+    <p>You've hit our 404 page. <a href="/">Go home</a>.</p>
```
Comment (sethwklein) [inline jsx/components/NoMatch.jsx]: Thou shalt "Go home"!

LUL

Comment (tylerconstance) [reply jsx/components/NoMatch.jsx]: Maybe I should add some styles to this page to give it a little more love visually.

**Comment by sethwklein** [comment]
LGTM 👍

**Pull request merged**